### ISSUE TITLE
aria-expanded is not used in a valid way

### ISSUE THREAD

**Issue by mikemarlett** [soflyy/oxygen-bugs-and-features#3296]
**Any bug report that doesn't include a Sandbox link or replication steps will be closed. If you can't provide these pieces of information, we're happy to help you via our official support channel instead: https://oxygenbuilder.com/support. You need to select "Something's broken and I need help" from the dropdown box on the left.**

**Describe the bug**
*A clear and concise description of what the bug is.*

aria-expanded is used in menus to give information about a menu state for accessibility. It is a binary setting and can only be true or false. Oxygen uses it to set some sort of internal data, and it needs to be either named something else or used only as the binary that is attended. Anything else is a WCAG 2.1 AA violation. (Which means that government and school sites cannot use Oxygen.)

Reference for proper usage:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

**A link to a Sandbox site where the bug has been reproduced https://oxygenbuilder.com/try? (REQUIRED)**
*Provide the link to a Sandbox install where the issue is present. If the issue only exists on a specific post or template, provide a direct link to that post or template.*

https://oxygen-qrbr78ph85dsl.oxygen-demo.qsandbox.org/product-tour/
See line 122 of the source code.

The Oxygen source code has aria-expanded in five places, and never uses it as true / false:

/component-framework/admin/pages.php:  line 409 (aria-expanded="settingsPopup")
/component-framework/components/classes/site-nav-class.php:  line 1073 (aria-expanded="RootOpen")
/component-framework/components/classes/site-nav-class.php:  line 1090  (aria-expanded="RootOpen")
/component-framework/components/classes/site-nav-class.php:  line 1163 (aria-expanded="open")
/component-framework/components/classes/site-nav-class.php:  line 1193 (aria-expanded="open")

**IF YOU CAN'T PROVIDE A SANDBOX REPRODUCTION: Steps required to reproduce the bug:**
1. Go to any template or default page or whatever. 
2. Add a site navigation menu 
3. Assign a menu
4. Save and go to the frontend
5. Search in the page source

**If you can't fill out the above information, you should not be filing a bug report. Instead, contact us via https://oxygenbuilder.com/support with the details you have and we'll investigate your issue further.**

**Comment by Kpudlo** [comment]
Thank you for the report. 

We use the JavaScript variables RootOpen and open, which both hold true or false values in the Site Navigation element. When you use a screen reader, the expanded states of submenus should be appropriately announced.
